test/bin/setup.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+set -o nounset
+set -o pipefail
+
+KUBECTL=${KUBECTL:-kubectl}
+KUSTOMIZE=kustomize
+
+project_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." ; pwd -P)"
+kubeconfig_dir=$project_dir/test/resources/kubeconfig
+hosting_cluster=${HOST_CLUSTER_NAME:-"hosting"}
+number=${CONTROLPLANE_NUMBER:-1}
+controlplane_image=${CONTROLPLANE_IMAGE:-"quay.io/open-cluster-management/multicluster-controlplane"}
+# Reuse certs will skip generate new ca/cert files under CERT_DIR
+# it's useful with PRESERVE_ETCD=true because new ca will make existed service account secrets invalided
+reuse_certs=${REUSE_CERTS:-false}
+certs_dir=${CERT_DIR:-"$project_dir/test/resources/controlplane/cert"}
+
+source "$project_dir/test/bin/util.sh"
+
+kube::util::ensure-gnu-sed
+kube::util::test_openssl_installed
+kube::util::ensure-cfssl
+
+printf "\033[0;32m%s\n\033[0m" "## Create KinD clusters"
+kind create cluster --name $hosting_cluster --image "kindest/node:v1.24.7" &
+for i in $(seq 1 "${number}"); do
+  kind create cluster --name controlplane$i-mc1 --image "kindest/node:v1.24.7" &
+done
+wait
+check_dir $kubeconfig_dir
+kind get kubeconfig --name $hosting_cluster > $kubeconfig_dir/$hosting_cluster
+for i in $(seq 1 "${number}"); do
+  kind get kubeconfig --name controlplane$i-mc1 > $kubeconfig_dir/controlplane$i-mc1
```

Comment:
You can specify `--kubeconfig $kubeconfig_dir/$hosting_cluster` in `kind create cluster` command

Reply:
Thx! I have added the `kubeconfig` flag when creating a KinD cluster.